frontend/src/components/EventDetailsList.tsx

```diff
@@ -0,0 +1,77 @@
+import {ListGroup} from "react-bootstrap";
+import {Action, PlantEventDetails} from "../model/calendar";
+import {Variant} from "react-bootstrap/types";
+
+
+function getInterval(action: Action) {
+    switch (action) {
+        case "fertilize":
+            return "weeks";
+        case "insolation":
```

Comment:
Czy w "insolations" chodzi o interwały? Mówiliśmy wcześniej o Enum typu "lubi słońce", "nie może być pod słońcem" itp

Reply:
Zgadza się, sama wartość insolation to będzie jakiś enum, ale możliwe że będą również jakieś eventy typu "przenieś roślinę na słońce". Ta funkcja zwraca tekst do wyświetlenia w przypadku opóźnienia w wykonaniu zadania, np. "2 weeks overdue". To czy w przypadku nasłonecznienia będziemy taki tekst wyświetlać jest jeszcze do ustalenia, ale na ten moment backend nie dostarcza żadnych eventów związanych z nasłonecznieniem, więc można zostawić to tak jak jest teraz.